plugins/modules/aci_aaa_certificate_authority.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross (@akinross) <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_aaa_certificate_authority
+short_description: Manage AAA Certificate Authorities (pki:TP)
+description:
+- Manage AAA Certificate Authorities on Cisco ACI fabrics.
+options:
+  name:
+    description:
+    - The name of the Certificate Authority.
+    type: str
+    aliases: [ certificate_authority, cert_authority, cert_authority_name, certificate_authority_name ]
+  description:
+    description:
+    - The description of the Certificate Authority.
+    type: str
+    aliases: [ descr ]
+  certificate_chain:
+    description:
+    - The PEM-encoded chain of trust from the trustpoint to a trusted root authority.
+    type: str
+    aliases: [ cert_data, certificate_data, cert, certificate ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+seealso:
+- module: cisco.aci.aci_aaa_certificate_authority
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(pki:TP).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Akini Ross (@akinross)
+"""
+
+EXAMPLES = r"""
+- name: Create a Certificate Authority
+  cisco.aci.aci_aaa_certificate_authority:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    name: example_authority
+    certificate_data: '{{ lookup("file", "pki/example_authority.crt") }}'
```

Comment:
```suggestion
    certificate_chain: '{{ lookup("file", "pki/example_authority.crt") }}'
```
Recommended to use the option name instead of an alias in the examples.